commands/version.go

```diff
@@ -0,0 +1,24 @@
+package commands
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(versionCmd)
+}
+
+var versionCmd = &cobra.Command{
+	Use:   "version",
+	Short: "Print the version number of Cartridge CLI",
+	Long:  `All software has versions. This is Cartridge CLI's`,
+	Args:  cobra.MaximumNArgs(0),
+	Run: func(cmd *cobra.Command, args []string) {
+		initLogger()
```

Comment:
I've found 2 mentions of "initLogger" function (excluding definition).

But seems that we should have only one entry point for that.

Reply:
Yes, thx.